README.md

```diff
@@ -221,16 +221,86 @@ root
 </pre>
 
 
-
-## Usage
-Todo
-
-### Compiling
-
-- Install go version > 1.8
-
-### Compiling
-
+## Getting Started
+The following section describes building the reference implementation of the Obscuro protocol, running the unit and 
+integration tests, and deploying a local testnet for end-to-end testing. The reference implementation of Obscuro is 
+written in [go](https://go.dev) with the IDE of choice being [GoLand](https://www.jetbrains.com/go/) from Jetbrains. 
+Unless otherwise stated, all paths stated herein are relative to the root of the `go-obscuro` checkout.
+
+
+### Dependencies
+The following dependencies are required to be installed locally;
+
+- [go](https://go.dev) (version > 1.8)
+- [docker](https://docs.docker.com/get-docker/) (recommend latest version)
+
+
+### Building
+To create the build artifacts local to the checkout of the repository the easiest approach is to build each component 
+separately for the host, enclave, and wallet extension  i.e. 
+
+```
+cd ./go/host/main && go build && cd -
+cd ./go/enclave/main && go build && cd -
+cd ./tools/walletextension/main && go build && cd -
```

Comment:
I'm not 100% we want this info for the obscuro node (host, enclave). As they are designed to run in docker and not bare metal.

Reply:
Yeah, that makes a lot of sense to me, thanks!

Reply:
Done